app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,31 @@
+class MoviesController < ApplicationController
+
+  def index
+    @movies = Movie.all
+      render :index, status: :ok
+  end
+
+  def show
+    @movie = Movie.find_by(id: movie_params[:id])
+    if @movie.nil?
+      render json: {
+        errors: {
+          id: ["No movie with ID #{movie_params[:id]}"]
+        }
+      }, status: :not_found
+    else
+      render :show, status: :ok
+    end
+  end
+
+  def create
+    @movie = Movie.new(movie_params)
+    @movie.save!
+    render :create, status: :ok
```

Comment:
Using bang (`!`) methods works great for the console or scripts like the seed file, but in your actual controller actions you should be calling the non-bang version and checking the result. As-is, this will produce an exception rather than a helpful error message, which is not a good user experience.

Something like this would be more appropriate:

```ruby
@movie = Movie.new(movie_params)
if @movie.save
  render :create, status: :ok
else
  render json: { errors: @movie.errors.messages }, status: :bad_request
end
```